src/main/resources/view/MainWindow.fxml

```diff
@@ -7,54 +7,48 @@
-         title="FoodRem" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
-    <icons>
-        <Image url="@/images/address_book_32.png"/>
-    </icons>
-    <scene>
-        <Scene>
-            <stylesheets>
-                <URL value="@DarkTheme.css"/>
-                <URL value="@Extensions.css"/>
-            </stylesheets>
+<fx:root xmlns:fx="http://javafx.com/fxml/1" type="javafx.stage.Stage"
+  xmlns="http://javafx.com/javafx/8" title="FoodRem" minWidth="450" minHeight="450" onCloseRequest="#handleExit">
+  <icons>
+    <Image url="@/images/address_book_32.png"/>
+  </icons>
+  <scene>
+    <Scene>
+      <stylesheets>
+        <URL value="@Extensions.css"/>
+        <URL value="@FoodRem.css"/>
+      </stylesheets>
 
-            <VBox>
-                <MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
-                    <Menu mnemonicParsing="false" text="File">
-                        <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit"/>
-                    </Menu>
-                    <Menu mnemonicParsing="false" text="Help">
-                        <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help"/>
-                    </Menu>
-                </MenuBar>
+      <VBox>
+        <!-- Menu Bar -->
+        <MenuBar fx:id="menuBar" VBox.vgrow="NEVER" styleClass="menu-bar">
+          <Menu text="Menu">
+            <MenuItem fx:id="helpMenuItem" onAction="#handleHelp" text="Help"/>
+            <MenuItem onAction="#handleExit" text="Exit"/>
+          </Menu>
+        </MenuBar>
 
-                <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
-                    <padding>
-                        <Insets top="5" right="10" bottom="5" left="10"/>
-                    </padding>
-                </StackPane>
-
-                <StackPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="pane-with-border"
-                           minHeight="100" prefHeight="100" maxHeight="100">
-                    <padding>
-                        <Insets top="5" right="10" bottom="5" left="10"/>
-                    </padding>
-                </StackPane>
-
-                <VBox fx:id="personList" styleClass="pane-with-border" minWidth="340" prefWidth="340"
-                      VBox.vgrow="ALWAYS">
-                    <padding>
-                        <Insets top="10" right="10" bottom="10" left="10"/>
-                    </padding>
-                    <StackPane fx:id="itemListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
-                </VBox>
-
-                <StackPane fx:id="statusbarPlaceholder" VBox.vgrow="NEVER"/>
+        <!-- Main Body -->
+        <HBox spacing="16" styleClass="main-body">
+          <!-- Left Column -->
+          <VBox spacing="12">
+            <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="command-placeholder"/>
```

Comment:
Maybe a user experience issue, but currently we would have to scroll right/left to see the full ItemCard (because teh fields are long). Should we do a "wrap text" within the card itself since this app should be optimized for keyboard usage rather than interacting with the GUI via a mouse

Reply:
Agreed, will fix this in the future PR. Let's close this MVP PR first to not block anyone.